src/musikcore/audio/PlaybackService.h

```diff
@@ -58,6 +58,7 @@ namespace musik { namespace core { namespace audio {
             /* our unique events */
             sigslot::signal2<size_t, musik::core::TrackPtr> TrackChanged;
             sigslot::signal0<> ModeChanged;
+            sigslot::signal0<> PlayPauseChanged;
```

Comment:
You shouldn't need this change -- if you look a few lines below below there's already a `PlaybackStateChanged` signal you can use instead. This signal emits following state changes: `Stopped`, `Paused`, `Prepared` (i.e. buffered, but not yet playing or paused -- this can happen in cases where the next playing track is optimistically pre-loaded), and `Playing`.

Reply:
Thanks for the helpful feedback, @clangen 🙏 The changes in 411980fe2ac15a018406c59b97a6c843bb21c320 use the existing `PlaybackStateChanged` instead of introducing a new `PlayPauseChanged` to signal playback state changes.